src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -115,14 +115,13 @@ func NewClient(ctx context.Context, opts ...gax.ClientOption) (*Client, error) {
     if err != nil {
         return nil, err
     }
-    return &Client {
+    c := &Client {
         conn: conn,
         client: google_example_library_v1.NewLibraryServiceClient(conn),
         callOptions: s.CallOptions,
-        metadata: map[string][]string{
-            "x-goog-api-client": []string{fmt.Sprintf("%s/%s %s gax/%s go/%s", s.AppName, s.AppVersion, gapicNameVersion, gax.Version, runtime.Version())},
-        },
-    }, nil
+    }
+    c.SetGoogleClientInfo("gax", gax.Version)
```

Comment:
Shouldn't this be the library's version? The gax's version is already "hard coded" into SetgoogleClientInfo.